Sources/Logger/Loggers/FileLogger/FileLogger.swift

```diff
@@ -11,15 +11,22 @@ import Zip
 enum FileLoggerError: Error {
     case missingWritableFileHandle
     case stringToDataConversionFailure
+    case userDefaultsInitFailure
+}
+
+public enum SharingConfiguration {
+    case shared(appGroupID: String)
```

Comment:
`// Sharing resources between the application and its extension. The user is required to set an App Group with assigned identifier. All shared resources (UserDefaults and FileManager) are then available for all applications within the group.`

Reply:
pridano